size-change-report.sh

```diff
@@ -94,7 +96,7 @@ function handle_spec {
 
     mapfile -t spec_a < <(tr ':' '\n' <<<"${spec}")
 
-    case "${spec_a}" in
+    case "${spec_a[0]}" in
```

Comment:
:D I've also seen that thanks to shellcheck, thanks for cleaning it up

Reply:
Oh yeah, shellcheck. Thanks for reminding me about the tool.